### PR TITLE
feat: all integer parameters should be sanitized toInt before usage

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -381,7 +382,8 @@ walletRouter.post('/send-tx',
         options: {
           min: 1
         }
-      }
+      },
+      toInt: true
     },
     inputs: {
       in: ['body'],
```
Comment (msbrogli) [inline index.js:389]: Don't we have to convert the `input[].index` to int as well?

File: index.js [hunk 2]
```diff
@@ -202,7 +202,7 @@ walletRouter.get(
  * For the docs, see api-docs.js
  */
 walletRouter.get('/address',
-  query('index').isInt().optional(),
+  query('index').isInt().optional().toInt(),
   query('mark_as_used').isBoolean().optional(),
```
Comment (msbrogli) [inline index.js:206]: `toBoolean()`? https://github.com/validatorjs/validator.js#sanitizers

File: index.js [hunk 3]
```diff
@@ -582,12 +584,12 @@ walletRouter.post('/melt-tokens',
...
+  query('max_utxos').isInt().optional().toInt(),
   query('token').isString().optional(),
   query('filter_address').isString().optional(),
-  query('amount_smaller_than').isInt().optional(),
-  query('amount_bigger_than').isInt().optional(),
-  query('maximum_amount').isInt().optional(),
+  query('amount_smaller_than').isInt().optional().toInt(),
+  query('amount_bigger_than').isInt().optional().toInt(),
+  query('maximum_amount').isInt().optional().toInt(),
   query('only_available').isBoolean().optional(),
```
Comment (msbrogli) [inline index.js:593]: `toBoolean()`? https://github.com/validatorjs/validator.js#sanitizers

**Review by msbrogli — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (pedroferreira1) [reply index.js:389]: Makes sense. Because of the way I had to validate the inputs, the conversion was a bit different.

File: index.js (hunk 2 above)
Comment (pedroferreira1) [reply index.js]: Done

File: index.js (hunk 3 above)
Comment (pedroferreira1) [reply index.js]: Done

**Review by msbrogli — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (msbrogli) [reply index.js:384]: Ok. Great!

**Review by thr0wn — APPROVED** [review]

**Pull request merged**